10.9-libcxx/stable/main/finkinfo/languages/sip-py.info

```diff
@@ -4,7 +4,7 @@ Info2: <<
 Package: sip-py%type_pkg[python]
 Version: 4.19.13
 Revision: 2
-Type: python (2.7 3.4 3.5 3.6 3.7)
+Type: python (2.7 3.4 3.5 3.6 3.7 3.8)
```

Comment:
For sip-pyXX-bin variants are mutually exclusive via Conflicts/Replaces. Adding py38 means that also needs to be added to the C/R entries of the SplitOff (and therefore bump Revision).

Reply:
Right, in #675 I had changed this to handle the binaries with `update-alternatives` for the update in 
738fdd9; forgot I had not pulled that over.

Reply:
Making any change to a Conflicts or Replaces field means the revision must be bumped.